src/views/News/index.jsx

```diff
@@ -25,6 +25,7 @@ import { isMobile } from 'react-device-detect';
 import Dropzone from 'react-dropzone';
 import newsService from 'services/news';
 import NewsList from './components/NewsList';
+import { userIsInAuthGroup } from './../../services/auth';
```

Comment:
You can use an absolute import to import from the services file
```suggestion
import { userIsInAuthGroup } from 'services/auth';
```